sdk/cosmosdb/cosmos/samples/v3/javascript/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@azure/cosmos": "latest",
     "dotenv": "latest",
-    "@azure/identity": "^2.0.1",
+    "@azure/identity": "^1.1.0",
```

Comment:
why are we lowering this version?

Reply:
The sample tool automatically picks the version of @azure/identity that @azure/cosmos uses for its own samples. We can just manually undo this change or upgrade @azure/cosmos to identity v2.

Reply:
@azure/cosmos Upgraded to identity v2